### PR TITLE
2608 - create template for anonymizer

### PR DESCRIPTION
**Pull request by shiranr**
Create template for anonymizer from cookiecutter and clean it up to our needs.

### CODE REVIEW

File: presidio-anonymizer/setup.cfg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[bdist_wheel]
+# This flag says to generate wheels that support both Python 2 and Python
+# 3. If your code will not run unchanged on both Python 2 and 3, you will
+# need to generate separate wheels for each Python version that you
+# support.
+universal=1
```
Comment (omri374) [inline presidio-anonymizer/setup.cfg:6]: This means that we can run on Python 2 which isn't the case.

**Comment by omri374** [comment]
Looks great! Thanks @shiranr

File: presidio-anonymizer/setup.cfg (hunk 1 above)
Comment (shiranr) [reply presidio-anonymizer/setup.cfg:6]: I thought I saw this in the analyzer as well so I assumed it's ok.

Comment (shiranr) [reply presidio-anonymizer/setup.cfg]: I removed it.

Comment (omri374) [reply presidio-anonymizer/setup.cfg]: You right! I'll remove it from there as well.

File: presidio-anonymizer/Pipfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+name = "pypi"
+
+[packages]
+flask = "==1.1.2"
+
+[dev-packages]
+pytest = "*"
+pylint = { version = "==2.3.1" }
+flake8= {version = "==3.7.9"}
+pytest-azurepipelines = "*"
```
Comment (omri374) [inline presidio-anonymizer/Pipfile:13]: Not necessary. We only use it during CI

**Review by omri374 — APPROVED** [review]

File: azure-pipelines.yml [hunk 3]
```diff
@@ -60,3 +60,22 @@ jobs:
...
         pipenv install --dev --skip-lock pytest-azurepipelines
         pipenv run pytest
+
+  - task: Bash@3
+    displayName: 'Linting: Anonymizer'
+    inputs:
+      targetType: 'inline'
+      workingDirectory: 'presidio-anonymizer'
+      script: |
+        pipenv run pylint presidio_anonymizer
```
Comment (SharonHart) [inline azure-pipelines.yml:70]: You can remove this line, we will use just flake8

File: presidio-anonymizer/Pipfile [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+flask = "==1.1.2"
+
+[dev-packages]
+pytest = "*"
+pylint = { version = "==2.3.1" }
```
Comment (SharonHart) [inline presidio-anonymizer/Pipfile:11]: remove pylint

**Review by navalev — APPROVED** [review]

File: presidio-anonymizer/setup.cfg [hunk 5]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 89
```
Comment (SharonHart) [inline presidio-anonymizer/setup.cfg:2]: Will have to be aligned with black, @omri374 are we going with a custom length different from PEP8 from a special reason?

**Review by SharonHart — APPROVED** [review]
Nice, left a small comment on pylint

**Review by SharonHart — DISMISSED** [review]

File: presidio-anonymizer/setup.cfg (hunk 5 above)
Comment (omri374) [reply presidio-anonymizer/setup.cfg:2]: Black uses 88 which is not aligned with PEP8 but makes more sense with modern monitors. This should also be 88 and not 89. Agree.

**Review by omri374 — APPROVED** [review]

File: presidio-anonymizer/setup.cfg (hunk 5 above)
Comment (shiranr) [reply presidio-anonymizer/setup.cfg:2]: @omri374  it should also be changed in the analyzer to 88. The value is currently similar in both.

File: azure-pipelines.yml [hunk 6]
```diff
@@ -60,3 +60,21 @@ jobs:
         # Install pytest and run tests
         pipenv install --dev --skip-lock pytest-azurepipelines
         pipenv run pytest
+
```
Comment (balteravishay) [inline azure-pipelines.yml:63]: Not sure this duplication is required. But can be fixed in the ci user story

**Review by balteravishay — DISMISSED** [review]
Approve with minor comment

**Pull request closed without merging**